backend/routers/projects.router.js

```diff
@@ -1,81 +1,38 @@
 const express = require("express");
 const router = express.Router();
 
-const Project = require("../models/project.model");
+const { ProjectController } = require('../controllers');
+const { verifyUser } = require('../middleware');
 
-// GET /api/projects/
-router.get("/", (req, res) => {
-  // const { headers } = req;
-  // const expectedHeader = process.env.CUSTOM_REQUEST_HEADER;
+// Display list of all Projects with GET.
+router.get('/', ProjectController.project_list);
 
-  // if (headers['x-customrequired-header'] !== expectedHeader) {
-  //     res.sendStatus(401);
-  // } else {
-  Project.find()
-    .then((projects) => {
-      if (!projects) {
-        res.json(false);
-      } else {
-        res.json(projects);
-      }
-    })
-    .catch((err) => {
-      console.log(err);
-      res.sendStatus(500).json({
-        message: `/GET Internal server error: ${err}`,
-      });
-    });
-  // };
-});
+// Create new Project with POST.
```

Comment:
Do we really need comments for all endpoints? As for me, they are self-explanatory.

Reply:
I don't think so, but I thought that I would follow with the convention that we had. I will remove them.